src/uwtools/cli.py

```diff
@@ -722,7 +727,7 @@ def _formatter(prog: str) -> HelpFormatter:
     """
     A standard formatter for help messages.
     """
-    return HelpFormatter(prog, max_help_position=8)
+    return HelpFormatter(prog, max_help_position=3)
```

Comment:
This was done to make the formatting consistent, since `"PATH"` was not long enough to trigger a new line.

Reply:
Does it make sense to add more info about this seemingly magic number, especially since it has consequences I wasn't even aware of?

Reply:
I've updated it to include an in-line comment explaining `max_help_position` and also updated the number to be 6, simply because I find it more aesthetically pleasing. However, I'm not sure why 6 (or even 7) is an acceptable number for a new line on`"PATH"`, while 8 is not.